src/Service/PropagateChanges.php

```diff
@@ -52,20 +68,20 @@ private function handleIndex(AbstractElement $element, IndexInterface $index): v
             return;
         }
 
-        $isPresent = $this->isIdInIndex($document::getElasticsearchId($element), $index);
+        $isPresent = $this->isIdInIndex($document::getElasticsearchId($element), $elasticaIndex);
 
         if ($document->shouldIndex($element)) {
             if ($isPresent) {
-                $this->updateElementInIndex($element, $index, $document);
+                $this->updateElementInIndex($element, $elasticaIndex, $document);
             }
 
             if (!$isPresent) {
-                $this->addElementToIndex($element, $index, $document);
+                $this->addElementToIndex($element, $elasticaIndex, $document);
             }
         }
 
         if ($isPresent && !$document->shouldIndex($element)) {
-            $this->deleteElementFromIndex($element, $index, $document);
+            $this->deleteElementFromIndex($element, $elasticaIndex, $document);
```

Comment:
This code does the job, but I think it could be optimised. Especially the `if($isPresent)` and `if(!isPresent)`.

What do you think about this suggestion?
```php
if ($isPresent) {
    $document->shouldIndex($element) ? $this->updateElementInIndex($element, $elasticaIndex, $document)
        : $this->deleteElementFromIndex($element, $elasticaIndex, $document);
} elseif ($document->shouldIndex($element)) {
    $this->addElementToIndex($element, $elasticaIndex, $document);
}
```